requirements.txt

```diff
@@ -0,0 +1,7 @@
+tensorflow
+mcap
+onnxruntime==1.16
+opencv-python
+numpy
+av
+git+https://github.com/MaivinAI/schemas.git@edgefirstDetect
```

Comment:
TODO: just a reminder that we need to merge and release the edgefirst-schemas before merging this PR.